src/IndexGame.py

```diff
@@ -6,14 +6,13 @@
 S10198398, Jeremiah Long
 
 '''
+#imports
+import gameinit
```

Comment:
May I know what is gameinit do?

Reply:
This just imports the function from gameinit to run gameinit.newgame in line 57